mail_layout_force/models/ir_ui_view.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2025 Quartile (https://www.quartile.co)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class IrUiView(models.Model):
+    _inherit = "ir.ui.view"
+
+    def generate_external_id(self):
+        self.ensure_one()
+        xml_id = f"view_{self.id}"
+        self.env["ir.model.data"]._update_xmlids(
+            [
+                {
+                    "xml_id": f"mail_layout_force.{xml_id}",
+                    "noupdate": True,
+                    "record": self,
+                }
+            ]
+        )
```

Comment:
We shouldn't be assigning an xml_id with `mail_layout_force`, or the substitute layout will be deleted when the module is uninstalled.
```suggestion
```